lib/SILOptimizer/Utils/CastOptimizer.cpp

```diff
@@ -573,6 +573,12 @@ CastOptimizer::optimizeBridgedSwiftToObjCCast(SILDynamicCastInst dynamicCast) {
     } else if (ConvTy.isExactSuperclassOf(DestTy)) {
       // The downcast from a base class to derived class may fail.
       if (isConditional) {
+        auto *FailureBB = dynamicCast.getFailureBlock();
```

Comment:
It looks like this `FailureBB` is shadowing the existing variable of the same name.

Reply:
I thought I sank it. I probably did it in a different patch from the one I cherry-picking.

Reply:
`FailureBB` is used below, so I can't tell if this code is correct.

Reply:
It was apart of a different cleanup commit that I didn't cherry-pick. I just merged it into this one.